packages/manager/src/components/DebouncedSearchTextField/DebouncedSearchTextField.tsx

```diff
@@ -7,12 +7,21 @@ import { InputAdornment } from 'src/components/InputAdornment';
 import { TextField, TextFieldProps } from 'src/components/TextField';
 import { usePrevious } from 'src/hooks/usePrevious';
 
-interface DebouncedSearchProps extends TextFieldProps {
+export interface DebouncedSearchProps extends TextFieldProps {
   className?: string;
+  /**
+   * Interaval of time that passes before search queries are accepted. Default is 400
+   */
```

Comment:
Spelling of `Interval ` and maybe the `@default` makes sense here?

```suggestion
  /**
   * Interval in milliseconds of time that passes before search queries are accepted.
   * @default 400
   */
```